### ISSUE TITLE
Attempting to deploy flutter game to iPhone6 

### ISSUE THREAD

**Issue by domesticmouse** [flutter/flutter#4269]
## Steps to Reproduce

Using the Atom.io Flutter tooling, attempt to run the flutter game on a device. Python blows up as follows:

```bash
flutter run --debug-port=16186 --start-paused --device-id 698b4da3275b3613ea0785a4b0718ce3ebf9bb38 • flutter logs
flutter_game/ • debug
Running lib/main.dart on Brett's iPhone...
[....] Waiting for iOS device to be connected
[....] Using iPhone 6 Plus 'Brett's iPhone' (698b4da3275b3613ea0785a4b0718ce3ebf9bb38).
------ Install phase ------
[  0%] Found iPhone 6 Plus 'Brett's iPhone' (698b4da3275b3613ea0785a4b0718ce3ebf9bb38) connected through USB, beginning install
[  5%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/META-INF/ to device
[  5%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/META-INF/com.apple.ZipMetadata.plist to device
[  6%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/_CodeSignature/ to device
[  6%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/_CodeSignature/CodeResources to device
[  7%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/app.flx to device
[ 12%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon29x29@2x.png to device
[ 12%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon29x29@2x~ipad.png to device
[ 13%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon29x29@3x.png to device
[ 14%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon29x29~ipad.png to device
[ 14%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon40x40@2x.png to device
[ 15%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon40x40@2x~ipad.png to device
[ 16%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon40x40@3x.png to device
[ 16%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon40x40~ipad.png to device
[ 17%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon60x60@2x.png to device
[ 18%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon60x60@3x.png to device
[ 18%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon76x76@2x~ipad.png to device
[ 19%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon76x76~ipad.png to device
[ 19%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/AppIcon83.5x83.5@2x~ipad.png to device
[ 20%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/embedded.mobileprovision to device
[ 21%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/ to device
[ 22%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/app.so to device
[ 23%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/ to device
[ 24%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/_CodeSignature/ to device
[ 24%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/_CodeSignature/CodeResources to device
[ 25%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/Flutter to device
[ 41%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/icudtl.dat to device
[ 44%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Frameworks/Flutter.framework/Info.plist to device
[ 44%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Info.plist to device
[ 45%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/LaunchScreen.storyboardc/ to device
[ 46%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/LaunchScreen.storyboardc/01J-lp-oVM-view-Ze5-6b-2t3.nib to device
[ 46%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/LaunchScreen.storyboardc/Info.plist to device
[ 47%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/LaunchScreen.storyboardc/UIViewController-01J-lp-oVM.nib to device
[ 48%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/PkgInfo to device
[ 48%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/Runner to device
[ 49%] Copying /Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app/ServiceDefinitions.json to device
[ 52%] CreatingStagingDirectory
[ 57%] ExtractingPackage
[ 60%] InspectingPackage
[ 60%] TakingInstallLock
[ 65%] PreflightingApplication
[ 65%] InstallingEmbeddedProfile
[ 70%] VerifyingApplication
[ 75%] CreatingContainer
[ 80%] InstallingApplication
[ 85%] PostflightingApplication
[ 90%] SandboxingApplication
[ 95%] GeneratingApplicationMap
[100%] Installed package ios/.generated/build/Release-iphoneos/Runner.app
------ Debug phase ------
Starting debug of iPhone 6 Plus 'Brett's iPhone' (698b4da3275b3613ea0785a4b0718ce3ebf9bb38) connected through USB...
[  0%] Looking up developer disk image
[ 95%] Developer disk image mounted successfully
[100%] Connecting to remote debug server
-------------------------
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python/lldb/__init__.py", line 98, in <module>
    import six
ImportError: No module named six
(lldb) command source -s 0 '/tmp/fruitstrap-lldb-prep-cmds-698b4da3275b3613ea0785a4b0718ce3ebf9bb38'
Executing commands in '/tmp/fruitstrap-lldb-prep-cmds-698b4da3275b3613ea0785a4b0718ce3ebf9bb38'.
(lldb)     platform select remote-ios --sysroot '/Users/brettmorgan/Library/Developer/Xcode/iOS DeviceSupport/9.3.2 (13F69)/Symbols'
  Platform: remote-ios
 Connected: no
  SDK Path: "/Users/brettmorgan/Library/Developer/Xcode/iOS DeviceSupport/9.3.2 (13F69)/Symbols"
(lldb)     target create "/Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
Current executable set to '/Users/brettmorgan/github/flutter_game/ios/.generated/build/Release-iphoneos/Runner.app' (arm64).
(lldb)     script fruitstrap_device_app="/private/var/containers/Bundle/Application/F775DAD5-2782-4383-92FC-6B3E7261CB8F/Runner.app"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'run_one_line' is not defined
error: python failed attempting to evaluate 'fruitstrap_device_app="/private/var/containers/Bundle/Application/F775DAD5-2782-4383-92FC-6B3E7261CB8F/Runner.app"'
```

I'm guessing we should add documentation somewhere on how to add six to the system python somewhere.

## Flutter Doctor

```bash
flutter doctor
[✓] Flutter (on Mac OS, channel master)
    • Flutter at /Users/brettmorgan/github/flutter
    • Framework revision 26a4d7bbad (60 minutes ago), engine revision b6ad88501f

[✓] Android toolchain - develop for Android devices (Android SDK 23.0.2)
    • Android SDK at /Users/brettmorgan/android-sdk-macosx
    • Platform android-23, build-tools 23.0.2
    • Java(TM) SE Runtime Environment (build 1.8.0_91-b14)

[✓] iOS toolchain - develop for iOS devices (Xcode 7.3.1)
    • XCode at /Applications/Xcode.app/Contents/Developer
    • Xcode 7.3.1, Build version 7D1014

[✓] Atom - a lightweight development environment for Flutter
    • flutter plugin version 0.2.2
    • dartlang plugin version 0.6.21
```

> For more information about diagnosing and reporting Flutter bugs, please see [https://flutter.io/bug-reports/](https://flutter.io/bug-reports/).

**Comment by abarth** [comment]
Crazy.  That's Apple's Python from `/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python/lldb/__init__.py`

If you use Xcode to create a "hello, world" program in Objective-C, do you get the same error?

**Comment by abarth** [comment]
> I'm guessing we should add documentation somewhere on how to add six to the system python somewhere.

We don't use Python ourselves.  That's Apple's code that's trying to import six.

**Comment by domesticmouse** [comment]
Oh gawd. I have a homebrew installed python on my path, that may be confusing things.

**Comment by eseidelGoogle** [comment]
https://github.com/flutter/flutter/issues/4245 appears to be the same issue.

**Comment by tvolkert** [comment]
Marking as dup of #4245

**Comment by cbracken** [comment]
Updated the fix and eliminated the doctor check in #19281.
